src/Agent.Worker/Container/DockerCommandManager.cs

```diff
@@ -308,6 +309,17 @@ public async Task<List<PortMapping>> DockerPort(IExecutionContext context, strin
             return DockerUtil.ParseDockerPort(portMappingLines);
         }
 
+        public async Task<bool> IsContainerRunning(IExecutionContext context, string containerId) {
+            List<string> filteredItems = await DockerPS(context, $"--filter id={containerId}");
```

Comment:
Could you please add some error handling?

Reply:
I've added error handling on Task Runner level. After catching an error it written warning message and continue execution of post-execution job without skipping